activestorage/test/analyzer/video_analyzer_test.rb

```diff
@@ -26,7 +26,7 @@ class ActiveStorage::Analyzer::VideoAnalyzerTest < ActiveSupport::TestCase
     assert_equal 480, metadata[:width]
     assert_equal 640, metadata[:height]
     assert_equal [4, 3], metadata[:display_aspect_ratio]
-    assert_equal 90, metadata[:angle]
+    assert_includes [90, -90], metadata[:angle]
```

Comment:
Easy way not to care about the version of ffmpeg? in the version < 5 this is 90 and 5.0+ it's -90